addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -106,28 +107,35 @@ Example item for the **indoor module**:
 Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home:inside:Co2" }
 ```
 
-**Supported channels for the indoor module:**
-
-* Temperature
-* TemperatureTrend
-* Humidity
-* Co2
-* Pressure
-* PressureTrend
-* AbsolutePressure
-* Noise
-* HeatIndex
-* Humidex
-* Dewpoint
-* DewpointDepression
-* WifiStatus
-* Location
-* TimeStamp
-* LastStatusStore
-* MinTemp
-* MaxTemp
-* DateMinTemp
-* DateMaxTemp
+**Supported channels for the main indoor module:**
+
+| Channel ID          | Item Type | Description                                              |
+|---------------------|-----------|----------------------------------------------------------|
+| Co2                 | Number    | Air Quality in ppm                                       |
+| Temperature         | Number    | Current temperature                                      |
+| TempTrend           | String    | Temperature Evolution Trend (up, down, stable)           |
+| Noise               | Number    | Current Noise Level in dB                                |
+| Pressure            | Number    | Current pressure in hPa                                  |
```

Comment:
You added the unit `hPa` here. Is this correct? wrt to `PressureUnit` channel.

Reply:
To be clear, I mainly copy/paste the description from the channels.
Of course, we can fix the channel descriptiuon too.

Reply:
Fair enough. I don't want to blame you for that. Just one thing I recognized during proofreading. It would be very much appreciated if you'll change it everywhere.